addons/mail/static/src/discuss/call/public/discuss_client_action_patch.js

```diff
@@ -25,4 +29,14 @@ patch(DiscussClientAction.prototype, {
             });
         }
     },
+    closeWelcomePage() {
+        super.closeWelcomePage(...arguments);
+        if (this.store.discuss_public_thread.defaultDisplayMode === "video_full_screen") {
+            const video = browser.localStorage.getItem("discuss_call_preview_join_video");
+            const mute = browser.localStorage.getItem("discuss_call_preview_join_mute");
+            this.rtc.toggleCall(this.store.discuss_public_thread, { video }).then(() => {
+                this.rtc.setMute(mute);
+            });
```

Comment:
Why is mute added afterwards?
Is there a time frame when mute is not taken into account?

Reply:
There is no "{boolean} audio" option to `toggleCall()`, and joining the call will so mute has to be made after the call started, since it's master I can do some changes to make it an option like `video`